### PR TITLE
Add CSS option to load countries package

### LINKED ISSUE #2929: Add alternative export for countries package
Explore alternative ways to render country symbols for faster render / easier usage, potential options
- add a direct (non-lazy version) country code to component map
- svg as inline css background image, hoping that images will render on a different render loop when used as cell renderer in ag grid

### PR DESCRIPTION
**Pull request by origami-z**
- Add `saltCountries.css` with SVG inline as background image

This helps loading speed of ag grid rows during scrolling, when used as custom cell renderer, compared to existing lazy loading.

Closes #2929

### CODE REVIEW

File: packages/countries/stories/CountrySymbol.qa.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { Meta, StoryFn } from "@storybook/react";
+import { StackLayout } from "@salt-ds/core";
+import { MX } from "@salt-ds/countries";
+import "@salt-ds/countries/saltCountries.css";
+
+export default {
+  title: "Country Symbols/Country Symbol/QA",
```
Comment (feliciackh) [inline packages/countries/stories/CountrySymbol.qa.stories.tsx:7]:
```suggestion
  title: "Country Symbols/Country Symbol/Country Symbol QA",
```
Nit

**Review by feliciackh — DISMISSED** [review]

**Review by feliciackh — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**